config/database.php

```diff
@@ -138,20 +148,27 @@
 
     'redis' => [
 
-        'client' => 'predis',
+        'client' => env('REDIS_CLIENT', 'phpredis'),
```

Comment:
A reason why this change?

Reply:
It comes from Laravel: https://github.com/laravel/laravel/pull/5085 and https://github.com/laravel/framework/pull/29688
predis is deprecated, in favor of phpredis.